From 46e8006d8d04ae7c3e45f0827e7775d6bdafb69e Mon Sep 17 00:00:00 2001
From: Johannes Pointner <johannes.pointner@gmail.com>
Date: Sat, 11 Nov 2017 15:02:18 +0100
Subject: [PATCH] Fix connection of PropertiesChanged event for LogindSeat

SDDM fails to start properly because the seat0 changes the property
canGraphical after SDDM is started and the connection to watch the
PropertiesChanged event didn't work.
---
 src/daemon/SeatManager.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/daemon/SeatManager.cpp b/src/daemon/SeatManager.cpp
index c5afc577..530977c7 100644
--- a/src/daemon/SeatManager.cpp
+++ b/src/daemon/SeatManager.cpp
@@ -40,7 +40,7 @@ namespace SDDM {
     Q_SIGNALS:
         void canGraphicalChanged(bool);
     private Q_SLOTS:
-        void propertiesChanged(const QString &interface, QVariantMap &changedProperties , const QStringList &invalidatedProperties);
+        void propertiesChanged(const QString &interface, const QVariantMap &changedProperties , const QStringList &invalidatedProperties);
     private:
         QString m_name;
         bool m_canGraphical;
@@ -80,7 +80,7 @@ namespace SDDM {
         return m_name;
     }
 
-    void LogindSeat::propertiesChanged(const QString& interface, QVariantMap& changedProperties, const QStringList& invalidatedProperties)
+    void LogindSeat::propertiesChanged(const QString& interface, const QVariantMap& changedProperties, const QStringList& invalidatedProperties)
     {
         Q_UNUSED(invalidatedProperties);
         if (interface != Logind::seatIfaceName()) {
